ios-drag-drop.js

```diff
@@ -221,16 +221,25 @@
           }
           return val;
         }.bind(this),
+        setDragImage: function(el, x, y){
+          this.customDragImage = el;
+          this.customDragImageX = x
+          this.customDragImageY = y
+        }.bind(this),
         dropEffect: "move"
       };
       this.el.dispatchEvent(evt);
     },
     createDragImage: function() {
-      this.dragImage = this.el.cloneNode(true);
-      
-      duplicateStyle(this.el, this.dragImage);
-      
-      this.dragImage.style.opacity = "0.5";
+      if (this.customDragImage) {
+        this.dragImage = this.customDragImage.cloneNode(true);
+        duplicateStyle(this.customDragImage, this.dragImage); 
+      } else {
+        this.dragImage = this.el.cloneNode(true);
+        duplicateStyle(this.el, this.dragImage); 
+        this.dragImage.style.opacity = "0.5";
```

Comment:
The opacity change should probably be after the if statement as it should also apply to custom drag images.